### PR TITLE
Making sure the bob class loader is used in the editor as well

### PR DESCRIPTION
**Pull request by JCash**

### CODE REVIEW

File: com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java [hunk 1]
```diff
@@ -479,19 +479,13 @@ else if (severity == MultipleCompileException.Info.SEVERITY_WARNING)
...
-        scanner.addUrl(new File(jar));
-    }
-
     public static ClassLoaderScanner getClassLoaderScanner() {
         return scanner;
     }
 
     private static void setupProject(Project project, boolean resolveLibraries, String sourceDirectory) throws IOException, LibraryException, CompileExceptionError {
-        createClassLoaderScanner();
+        scanner = Project.createClassLoaderScanner();
```
Comment (JCash) [inline com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java:487]: Note: The editor doesn't really invoke "bob", but a manually created instance of the Project class.

File: com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java [hunk 2]
```diff
@@ -172,6 +172,14 @@ public Publisher getPublisher() {
         return this.publisher;
     }
 
+    public static ClassLoaderScanner createClassLoaderScanner() throws IOException {
+        // Find the jar file in the built-in resources
+        String jar = Bob.getJarFile("fmt-spine.jar");
+        ClassLoaderScanner scanner = new ClassLoaderScanner();
+        scanner.addUrl(new File(jar));
+        return scanner;
+    }
```
Comment (JCash) [inline com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java:181]: Moving the functionality to the Project class

File: editor/src/clj/editor/pipeline/bob.clj [hunk 3]
```diff
@@ -128,7 +128,7 @@
         (doseq [[key val] bob-args]
           (.setOption bob-project key val))
         (.setOption bob-project "liveupdate" (.option bob-project "liveupdate" "no"))
-        (let [scanner (ClassLoaderScanner.)]
+        (let [scanner (^ClassLoaderScanner Project/createClassLoaderScanner)]
```
Comment (JCash) [inline editor/src/clj/editor/pipeline/bob.clj:131]: Use the proper custom class loader.

**Review by britzl — APPROVED** [review]
Looks good!

**Pull request merged**